### PR TITLE
Update segment snippet script to latest version

### PR DESCRIPTION
**Pull request by jkeen**
Updating this snippet makes this (apparently) work correctly with ember-cli 2.0.2

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,10 +10,9 @@ module.exports = {
...
-                  'analytics.load("' + config.segment.WRITE_KEY + '");' +
+                '!function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","page","once","off","on"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t){var e=document.createElement("script");e.type="text/javascript";e.async=!0;e.src=("https:"===document.location.protocol?"https://":"http://")+"cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";var n=document.getElementsByTagName("script")[0];n.parentNode.insertBefore(e,n)};analytics.SNIPPET_VERSION="3.1.0";' +
+                  'analytics.load("' + config.segment.WRITE_KEY + '"); analytics.page()' +
```
Comment (josemarluedke) [inline index.js]: The addon calls automatically `analytics.page()`, so no need for it.

**Comment by jkeen** [comment]
@josemarluedke Updated.

**Comment by josemarluedke** [comment]
Awesome @jkeen! Thank you.

**Comment by josemarluedke** [comment]
Just pushed v0.0.8 to npm.

**Pull request merged**